src/CaBundle.php

```diff
@@ -105,15 +100,18 @@ public static function getSystemCaRootBundlePath(LoggerInterface $logger = null)
             '/usr/local/etc/openssl/cert.pem', // OS X homebrew, openssl package
         );
 
+        foreach($otherLocations as $location) {
+            $otherLocations[] = dirname($location);
+        }
```

Comment:
This will make it use any of these random dirs if nothing else matched, right? That sounds suboptimal to me, I think we should rather not do this.

Reply:
I agree, but this matches the current behaviour

Reply:
Oh true, never mind then, hadn't looked at this in a while :)